src/features/Courses/CourseDetailTable/columns.jsx

```diff
@@ -111,6 +89,82 @@ const columns = [
     accessor: 'endDate',
     Cell: ({ row }) => (row.values.endDate ? formatUTCDate(row.values.endDate, 'MM/dd/yy') : '-'),
   },
+  {
+    Header: '',
+    accessor: 'courseName',
+    cellClassName: 'dropdownColumn',
+    disableSortBy: true,
+    Cell: ({ row }) => {
+      const {
+        masterCourseName,
+        masterCourseId,
+        classId,
+        className,
+        startDate,
+        endDate,
+        minStudentsAllowed,
+        maxStudents,
+      } = row.original;
+
+      const [isOpenModal, openModal, closeModal] = useToggle(false);
+
+      return (
+        <Dropdown className="dropdowntpz">
+          <Dropdown.Toggle
+            id="dropdown-toggle-with-iconbutton"
+            as={IconButton}
+            src={MoreHoriz}
+            iconAs={Icon}
+            variant="primary"
+            data-testid="droprown-action"
+            alt="menu for actions"
+          />
+          <Dropdown.Menu>
+            <Dropdown.Item>
+              <a
+                href={`${getConfig().LEARNING_MICROFRONTEND_URL}/course/${classId}/home`}
```

Comment:
This link is not working

Reply:
What do u mean about "is not working" the url is wrong or the parameters are bad?

Reply:
It doesn't open the link:
![drowdownBug](https://github.com/Pearson-Advance/frontend-app-institution-portal/assets/36944773/a1119445-4dbc-4d7b-a3a5-655401c1a0d5)

Reply:
🫨